ramesh/data/Lab-2-stages-of-data-visualization.ipynb

```diff
@@ -0,0 +1,842 @@
+{
```

Comment:
<p>What columns are relevant and why?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/suparthghimire/data-analysis-and-visualization-bca-2025/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>